feature-libs/quote/components/quote-list/quote-list.component.html

```diff
@@ -59,7 +60,7 @@
                 {{ quote?.name }}
```

Comment:
Quote is defined here
```suggestion
                {{ quote.name }}
```